sdk/datashare/Microsoft.Azure.Management.DataShare/src/Microsoft.Azure.Management.DataShare.csproj

```diff
@@ -5,11 +5,11 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataShare</PackageId>
     <Description>Azure Data Share is a first party resource provider that enables customers to easily and securely share big data with other organizations in Azure, with centralized management, monitoring and governance. </Description>
-    <Version>0.9.0-preview</Version>
+    <Version>1.0.0-preview</Version>
```

Comment:
Can you please make this `1.0.0-preview.1`? If there is another preview it will go to `1.0.0-preview.2`

Reply:
updated with 1.0.0-preview.1